### ISSUE TITLE
Changing gotoh directory name broke directory path 

### ISSUE THREAD

**Issue by ewong347** [PoonLab/covizu#17]
From nohup/cronlog: 
> Traceback (most recent call last):
>   File "/home/covid/gotoh2.py", line 262, in <module>
>     aligner = Aligner()
>   File "/home/covid/gotoh2.py", line 25, in __init__
>     files = pkgres.resource_listdir('gotoh2', 'models')
>   File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 1190,$
>     resource_name
>   File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 1486,$
>     return self._listdir(self._fn(self.module_path, resource_name))
>   File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 1574,$
>     return os.listdir(path)
> OSError: [Errno 2] No such file or directory: '/home/covid/models'

**Comment by ArtPoon** [comment]
Looks like it's still trying to run the gotoh2 package from the local directory instead of the default Python package installation directory..

**Comment by ewong347** [comment]
Hacky solution was to edit lines 25, 29 from "gotoh" -> "gotoh-src" in local files.

**Comment by ArtPoon** [comment]
This isn't acceptable - we need to figure out a more permanent fix.

**Comment by ArtPoon** [comment]
Can you please post a minimal reproducible example?

**Comment by ewong347** [comment]
ewong347@Langley:/home/covid$ python gotoh2.py --append test.fa GISAID-2020-05-15.fasta NC_045512.fa

> 
> Traceback (most recent call last):
>   File "gotoh2.py", line 262, in <module>
>     aligner = Aligner()
>   File "gotoh2.py", line 25, in __init__
>     files = pkgres.resource_listdir('gotoh2', 'models')
>   File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 1190, in resource_listdir
>     resource_name
>   File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 1486, in resource_listdir
>     return self._listdir(self._fn(self.module_path, resource_name))
>   File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 1574, in _listdir
>     return os.listdir(path)
> OSError: [Errno 2] No such file or directory: '/home/covid/models'
>

**Comment by ArtPoon** [comment]
Ok, the problem was having a script in the directory named `gotoh2.py`.  Seems to be a self-referencing problem.  Use this:
```console
art@Langley:/home/covid$ python update.py -ref NC_045512.fa GISAID-2020-05-15.fasta test.fa
Aligning new sequence hCoV-19/USA/IL-NM06/2020|EPI_ISL_444520|2020-03-17
```

**Comment by ArtPoon** [comment]
Updated gotoh2 installation on Langley to latest commit (with changes from @RouxCil merged in - `e7605993f2f6fd0ae05f317a3903ec7f83cdd9ad`)
